Tmedia 741 byline global phrase

JackHowa opened the pull request:
- Move translations in lokalise from byline to global phrases 
- Run generate intl to generate the block changes
- goes with @malavikakoppula 's large promo #1411

JackHowa committed:
feat: Add global phrase byline block. Include other tab changes

JackHowa committed:
chore: Update byline styles and global phrases

JackHowa committed:
chore: Update other languages than english for byline and global phrases

JackHowa committed:
chore: Update all translations based on lokalise

JackHowa commented on blocks/overline-block/intl.json:
```diff
@@ -1,13 +1,13 @@
 {
 	"overline-sponsored-content": {
-		"de": "Werblicher Inhalt",
+		"de": "Sponsored Content",
```
interesting this looks like a regression

matthewroach replied:
We can take the existing values and add to lokalise then regenerate the locale files to avoid this regression

matthewroach replied:
We need to fix these before merging

JackHowa replied:
oh ok

JackHowa replied:
it seems like lokalise may have an issue ... trying to see what's happened there

JackHowa replied:
cool can manually update lokalise with this

JackHowa replied:
updated to overline

matthewroach commented on blocks/byline-block/intl.json:
```diff
@@ -1,22 +0,0 @@
-{
```
Removing this file will require updates to the code in this block to ensure it's pull the translation by the new key names now

JackHowa replied:
updated

matthewroach replied:
I pushed an updated - as the byline block intl.json file is manually curated and the values are not part of any locale main file

JackHowa replied:
nice

matthewroach commented on blocks/shared-styles/intl.json:
```diff
@@ -21,28 +21,6 @@
 		"pt": "Vídeo",
 		"sv": "Video"
 	},
-	"byline.by-text": {
```
This will mean current blocks (not migrated) will not have any translations for these phrases at the moment

JackHowa replied:
oh I thought that's what we wanted. will update

matthewroach replied:
Yeah - that's fine by me, but may cause some confusion when a developer comes to do a migration of a block still using code from Shared Styles - technically there are still consumers of this code until the last block is converted

JackHowa replied:
added back

JackHowa committed:
fix: Add back byline shared styles

JackHowa committed:
chore: Remove extra newline

JackHowa committed:
test: Update translations so they're not tied to intl

JackHowa committed:
test: Update to include itnl change

JackHowa committed:
fix: Revert identity change

JackHowa committed:
fix: Update connected as issue that was not implemented in code and still in lokalise

JackHowa commented:
hopefully this will resolve @nschubach and @malavikakoppula's work

matthewroach committed:
Update byline translations

The byline block intl.json file is manually curated and not part of our locales, switched to use the global phrases

matthewroach reviewed (dismissed):
We need to update locale to avoid the sponsored content regression

nschubach commented:
> We need to update locale to avoid the sponsored content regression

Are you referring to this which I added to `extra-large-promo-block`?
https://github.com/WPMedia/arc-themes-blocks/blob/f67cf98c4dc14f5d0e46ffa2a81fc00d36132f5d/blocks/global-phrases-block/intl.json#L134

matthewroach commented:
@nschubach - Yes, if we have those translation available we should be setting them in their respective locations correctly

JackHowa committed:
fix: Update overline sponsored content

JackHowa committed:
Merge branch 'TMEDIA-741-byline-global-phrase' of github.com:WPMedia/fusion-news-theme-blocks into TMEDIA-741-byline-global-phrase

JackHowa committed:
feat: Add the global sponsored content and remove the overline specific blocks translations

Commit:
Merge branch 'arc-themes-release-version-2.0.1' into TMEDIA-741-byline-global-phrase

JackHowa commented:
@matthewroach feedback around sponsored content has been addressed. not sure how that happened on lokalise's side. updated to global phrases

nschubach commented on blocks/subscriptions-block/intl.json:
```diff
@@ -77,15 +77,15 @@
-		"no": "Connected as %{email}",
-		"pt": "Connected as %{email}",
-		"sv": "Connected as %{email}"
+		"de": "Connected",
+		"en": "Connected",
+		"es": "Connected",
+		"fr": "Connected",
+		"ja": "Connected",
+		"ko": "Connected",
+		"no": "Connected",
+		"pt": "Connected",
+		"sv": "Connected"
```
You said this broke something?

It's passed in Subs, ContactInfo: 
![image](https://user-images.githubusercontent.com/2287238/180065664-aa791417-d630-4dab-ad13-0595a0bfe4ce.png)

nschubach replied:
not sure it's necessary as is so I don't care either way.

nschubach commented on locale/de.json:
```diff
@@ -1,448 +1,447 @@
+		"product-assortment-carousel.left-arrow-label": "Previous",
+		"product-assortment-carousel.slide-indicator": "Slide %{current} of %{maximum}"
+	}
+}
```
```suggestion
}

```

Commit:
Update locale/de.json

Co-authored-by: nschubach <nschubach@gmail.com>
